src/scraper_auchan/products_spider.py

```diff
@@ -88,19 +88,20 @@ def extract_eans(response) -> list[str]:
                 return eans
 
     @staticmethod
-    def extract_price(response) -> float:
+    def extract_price(response, base=True) -> float:
```

Comment:
I think the `extract_price` function should be reworked to explicitly indicate which price (the base price or the discounted one) is returned. Optionally, multiple values could be returned at once.

In the current situation, if a product isn't discounted, the same value is stored in the database in both `amount` and `discounted_amount` fields. The `discounted_amount` database field being optional, no value should be set when no discount is detected.

Reply:
Still the case. Why is it marked as resolved?